dashboard/config/linux/bits/base.yml

```diff
@@ -206,6 +206,9 @@ config:
  # Don't test/need this.
  - XEN: n
 
+ # Let's keep it enabled even for baseline builds as otherwise it blocks GENDWARFKSYMS and RUST.
+ - MODVERSIONS: [-allyes]
```

Comment:
MODVERSIONS migrated here from `sybsystems`.
`subsystems` didn't include it because android-6.12 has `onlyusb` label.
If you need it in rust.yml and rust seems to be the only reason to have it, why do you think `base.yml` is a better place?

I can't understand the commend above.